.grabl/automation.yml

```diff
@@ -20,7 +20,7 @@ build:
     build:
       machine: graknlabs-ubuntu-20.04
       script: |
-        bazel build //... --test_output=errors
+        bazel build --config=rbe //... --test_output=errors
         bazel run @graknlabs_dependencies//tool/checkstyle:test-coverage
-        bazel test $(bazel query 'kind(checkstyle_test, //...)')
+        bazel test --config=rbe $(bazel query 'kind(checkstyle_test, //...)') --test_output=errors
       
```

Comment:
can you add a new line here?

Reply:
Hmm, where?

Reply:
at the end of the file. There's no empty line. But I can also just merge this PR first and you do that in a subsequent PR.